hasura/graphql-engine
# Set LANG=C.UTF-8 in the docker container (fix #2571)

### Description

This is a quick fix for #2571 that incorporates the changes from hasura/haskell-docker-builder#1 and sets `LANG=C.UTF-8` in the `hasura/graphql-engine` image. I have some bigger changes on another branch that change the CI around in more drastic ways to try and make it possible to write tests for this kind of thing, but that’s proven to be a lot of work, so this is a simple solution for now.

### Affected components 


- Server
- Build System

### Related Issues

#2571, hasura/haskell-docker-builder#1

### Steps to test and verify

Since this isn’t under test coverage, it needs to be tested manually: run the image against a database with some Unicode text in it (such as the schema with a comment mentioned in #2571) and make sure hitting the `pg_dump` endpoint works okay.
